cmd/eval.go

```diff
@@ -626,6 +628,10 @@ func setupEval(args []string, params evalCommandParams) (*evalContext, error) {
 		regoArgs = append(regoArgs, rego.Capabilities(params.capabilities.C))
 	}
 
+	if params.strict {
+		regoArgs = append(regoArgs, rego.Strict(true))
+	}
```

Comment:
[nit] Unless I'm missing something, this is equivalent to
```go
regoArgs = append(regoArgs, rego.Strict(params.strict))
```

But this is OK, too.